core/os/time.cpp

```diff
@@ -263,6 +278,11 @@ String Time::get_time_string_from_unix_time(int64_t p_unix_time_val) const {
 
 Dictionary Time::get_datetime_dict_from_datetime_string(String p_datetime, bool p_weekday) const {
 	PARSE_ISO8601_STRING(Dictionary())
+	VALIDATE_YMDHMS()
+	const_cast<String &>(datetime_error) = validation_error;
+	if (validation_error != "") {
```

Comment:
I'm not sure if we actually practice this in the engine anywhere, but IMO variables that appear out of nowhere aren't a great idea.

Reply:
Sheesh, that's much better. I didn't know you could simply define it in the outer scope. Thanks.